src/ripple/app/tx/impl/apply.cpp

```diff
@@ -115,6 +115,7 @@ apply (Application& app, OpenView& view,
     STTx const& tx, ApplyFlags flags,
         beast::Journal j)
 {
+    disableUnderflowFix(view.info().parentCloseTime);
```

Comment:
I'd like to return the underflow fix to it's original value. How about an RAII class that in the constructor takes the close time, saves the old value, sets the new value, and in the destructor restores the old value?